yarn.lock

```diff
@@ -1048,6 +1048,18 @@
   dependencies:
     "@emotion/memoize" "^0.6.6"
 
+"@emotion/is-prop-valid@^0.7.3":
+  version "0.7.3"
+  resolved "https://registry.yarnpkg.com/@emotion/is-prop-valid/-/is-prop-valid-0.7.3.tgz#a6bf4fa5387cbba59d44e698a4680f481a8da6cc"
+  integrity sha512-uxJqm/sqwXw3YPA5GXX365OBcJGFtxUVkB6WyezqFHlNe9jqUWH5ur2O2M8dGBz61kn1g3ZBlzUunFQXQIClhA==
+  dependencies:
+    "@emotion/memoize" "0.7.1"
+
+"@emotion/memoize@0.7.1":
```

Comment:
Could we remove the emotion/memoize@^0.6.1 and 0.6.6 deps below and only have one install of memoize?

Reply:
Why doesn't yarn resolve this itself :/

Reply:
If you delete all the `@emotion/memoize` blocks in the yarn.lock file and re-run `yarn install`, yarn can install the dependencies with tighter resolution.

Reply:
Unfortunately it looks like these won't tighten up since we ask for `is-prop-type: ^0.7.3` in `react-pose` and then ask for `is-prop-type: ^0.6.1` in `create-emotion-styled`.